content/en/docs/guides/custom-name-resolution.md

```diff
@@ -0,0 +1,104 @@
+---
+title: Custom Name Resolution
+description: >-
+  Explains standard name resolution, the custom name resolver interface,
+  and how to write an implementation.
+---
+
+### Overview
+
+Name resolution is fundamentally about service discovery. When sending a request
+to `grpc.io`, the client must determine the IP address of that service name.
+Name resolution is often thought to be the same as
+[DNS](https://www.ietf.org/rfc/rfc1035.txt). In practice however, DNS is usually
+augmented with extensions or completely replaced to enable name resolution.  For
+example, the `/etc/hosts` file on Posix systems is not strictly speaking part of
+DNS and instead stems from
+[RFC-952](https://datatracker.ietf.org/doc/html/rfc952).
+
+When making a request with a gRPC client, by default, DNS name resolution is
```

Comment:
Only DNS A records right? Do we support DNS SRV?

Reply:
Only with grpclb, which is deprecated, so I think this is accurate.